src/components/boards-bar/boards-bar.module.css

```
@@ -5,13 +5,30 @@
+  background: transparent;
+  border: 0;
+  color: inherit;
+  display: inline;
+  font: inherit;
+  line-height: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+.boardNavDesktop button:focus-visible, .boardNavMobile button:focus-visible {
+  outline: 1px dotted currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)